src/tooltip/tooltip.ts

```diff
@@ -83,11 +62,11 @@ export class NgbTooltip implements OnInit, OnDestroy {
   @Input() autoClose: boolean | 'inside' | 'outside';
   /**
     * Placement of a tooltip accepts:
+    *  - Array or space separated string of following values:
     *    "top", "top-left", "top-right", "bottom", "bottom-left", "bottom-right",
-    *    "left", "left-top", "left-bottom", "right", "right-top", "right-bottom"
-    * and array of above values.
+    *    "left", "left-top", "left-bottom", "right", "right-top", "right-bottom",
     */
-  @Input() placement: PlacementArray;
+  @Input() placement: string | PlacementArray;
```

Comment:
Let's open a separate feature PR for this later for ALL widgets supporting positioning